tst/testinstall/IsLIBGAP.tst

```diff
@@ -0,0 +1,12 @@
+#############################################################################
+##
+#W  IsLIBGAP.tst                  GAP library               Sebastian Gutsche
+##
+##
+#Y  Copyright (C)  2018,  GAP Group
+##
+gap> START_TEST("IsLIBGAP.tst");
+gap> IsBound( IsLIBGAP );
+true
```

Comment:
You may want to run `./gap tst/testinstall.g` before submitting an updated version of this PR, to make sure it really works.